### PR TITLE
Home work 17

### PR DESCRIPTION
**Pull request by reemaRahhal**
I have been finished my task ,  I searched in  google for valid email pattern then I wrote a function and did a lot of testing.
please, can you reviwe my task

### CODE REVIEW

File: Home_Work_17/js/script.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+ function validEmail() {
```
Comment (EliasSalom) [inline Home_Work_17/js/script.js:1]: this function has been used before and copied before please change it to a new one

**Review by EliasSalom — CHANGES_REQUESTED** [review]
please check my comments and fix the issues!!

File: Home_Work_17/js/script.js (hunk 1 above)
Comment (reemaRahhal) [reply Home_Work_17/js/script.js]: I had changed the function.

**Review by EliasSalom — APPROVED** [review]
Good work

**Pull request merged**